generator/src/OperatorTestGenerator.php

```diff
@@ -138,11 +138,11 @@ private function convertYamlTaggedValues(mixed $object): mixed
             $value = $object->getValue();
 
             return match ($object->getTag()) {
-                'regex' => new Regex(...(array) $value),
-                'long' => new Int64($value),
-                'double' => new Decimal128($value),
-                'date' => new UTCDateTime($value),
-                'binary' => new Binary(base64_decode($value)),
+                'bson_regex' => new Regex(...(array) $value),
+                'bson_long' => new Int64($value),
+                'bson_double' => new Decimal128($value),
```

Comment:
If we're prefixing these with `bson_`, can we use the actual names for clarity?

That would be `bson_int64` above, `bson_decimal128` here, and `bson_utcdatetime` below.

Reply:
Ok, renamed.